.github/workflows/main.yml

```diff
@@ -140,6 +131,9 @@ jobs:
           docker cp $GITHUB_WORKSPACE/l10n singleton:/
           cd $GITHUB_WORKSPACE
           cp -r $GITHUB_WORKSPACE/devops/autotest/client/ruby/l10n/bundles/* $GITHUB_WORKSPACE/l10n/bundles
+          cd $GITHUB_WORKSPACE
+          gem build singleton-client.gemspec
```

Comment:
Why does it need to build gem?

Reply:
Hi Xiaochao, yes, for the smoke test project, it needs to import the "gem build" generated gem.